package.json

```diff
@@ -36,10 +36,13 @@
   "bin": {
     "lirc_web": "./app.js"
   },
+  "config": {
+    "test_env": true
```

Comment:
Should this be set to false by default?

Reply:
as far as I have understood, this is applied only on calls of the script section. There we only have test calls and this default is exactly what we need.